data/events.yaml

```diff
@@ -98,3 +98,109 @@ x-events:
   severity: "info"
   properties:
     cluster_id: UUID
+
+- name: cluster_installing_finished
+  message: "Successfully finished installing cluster {cluster_name}"
```

Comment:
Honestly I think that the `cluster_name` can be removed from all the events. It's irrelevant to all of them... Currently the events have different syntax and sometimes the name appears as a prefix, as a suffix, or even in the middle of the message.
WDYT? @filanov @avishayt

Reply:
I agree that it can be removed from all the events where the cluster-ID is the "primary key", which is almost all of the current events.  The user is listing events by cluster-ID in this case so we assume that they know the cluster name.  I would keep it only for events related to late binding (host bound to cluster <name>, or unbound from cluster <name>).

Reply:
done, is this comment relevant also for host-type events with `host_name`? @YuviGold @avishayt  (asking for following PRs)

Reply:
The host name is useful since the user usually looks at all events related to a cluster and its hosts.  However, we should keep the format consistent, as @YuviGold suggested.  Perhaps something like: "Host <name> <verb> <information>".

Reply:
BTW the consistent formatting would probably be a separate task.